src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -162,6 +174,29 @@ impl VirtSandbox {
         })
     }
 
+    async fn prepare_rootfs_config(&self) -> Result<BlockConfig> {
+        let hypervisor_config = self.hypervisor.hypervisor_config().await;
+
+        let image = {
+            let initrd_path = hypervisor_config.boot_info.initrd.clone();
+            let image_path = hypervisor_config.boot_info.image;
+            if !initrd_path.is_empty() {
+                Ok(initrd_path)
+            } else if !image_path.is_empty() {
+                Ok(image_path)
+            } else {
+                Err(anyhow!("failed to get image"))
+            }
+        }
+        .context("get image")?;
+
+        Ok(BlockConfig {
+            path_on_host: image,
+            is_readonly: true,
+            ..Default::default()
+        })
```

Comment:
Does it need to specify the vm rootfs driver?

Reply:
thx, fixed